mycompany-seeq-link-connector-myconnector/src/main/java/com/mycompany/seeq/link/connector/DatasourceSimulator.java

```diff
@@ -32,6 +37,30 @@ public boolean getStepped() {
         }
     }
 
+    public class TagValue {
```

Comment:
I think we should be very explicit (not intended for production) and add some more documentation here